week-1/errors/3.js

```diff
@@ -1,7 +1,13 @@
 const cardNumber = 4533787178994213;
-const last4Digits = cardNumber.slice(-4);
+const last4Digits = cardNumber.toString().slice(-4);
```

Comment:
Good - always remember that in a larger application, you would need to be careful when changing types like this - another part of the code might expect an integer type and behave oddly if you change it.